### ISSUE TITLE
[data request] MVTec AD : Image Anomaly Detection

### ISSUE THREAD

**Issue by innat** [tensorflow/datasets#3956]
* Name of dataset: MVTec AD
* URL of dataset: https://www.mvtec.com/company/research/datasets/mvtec-ad
* License of dataset: Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License (CC BY-NC-SA 4.0)
* Dataset Paper: [CVPR-LINK](https://openaccess.thecvf.com/content_CVPR_2019/papers/Bergmann_MVTec_AD_--_A_Comprehensive_Real-World_Dataset_for_Unsupervised_Anomaly_CVPR_2019_paper.pdf).
* Short description of the dataset and use case(s): MVTec AD is a dataset for benchmarking anomaly detection methods focusing on industrial inspection. It contains over 5000 high-resolution images divided into fifteen object and texture categories. Each category comprises a set of defect-free training images and a test set of images with various kinds of defects and images without defects.

---

Folks who would also like to see this dataset in `tensorflow/datasets`, **please thumbs up** so the developers can know which requests to prioritize.

![164736543-0ef58c66-dfb0-47c5-8e6e-26774dbc6fd3](https://user-images.githubusercontent.com/17668390/170335115-aae672d1-3ed1-4d85-a6a9-70fcbee88713.gif)
(above gif from [here](https://www.kaggle.com/code/ipythonx/mvtec-ad-anomaly-detection-with-anomalib-library), inference output of a model on this dataset.)

**Note**

- AFAIK, this dataset is one of the most well-used and high-quality datasets for this task.
- AFAIK, there is no similar or equivalent dataset available in `tensorflow/dataset`. 
- AFAIK, the overall goal (image anomaly detection) is less practiced with `tensorflow` due to limited resources. ( Mostly contained auto-encoder modeling stuff. )
- AFAIK, all SOTA models (in PyTorch) are primarily benchmarked on this dataset. [SOURCE](https://paperswithcode.com/sota/anomaly-detection-on-mvtec-ad).

**Comment by innat** [comment]
@Conchylicultor any feedback?
